skimage/measure/fit.py

```diff
@@ -668,7 +668,7 @@ def _dynamic_max_trials(n_inliers, n_samples, min_samples, probability):
         return np.inf
     inlier_ratio = n_inliers / n_samples
     nom = max(_EPSILON, 1 - probability)
-    denom = max(_EPSILON, 1 - inlier_ratio ** min_samples)
+    denom = max(_EPSILON, 1 - inlier_ratio ** min_samples - _EPSILON)
```

Comment:
```suggestion
    denom = max(_EPSILON, 1 - inlier_ratio ** min_samples)
    denom = min(denom, 1 - _EPSILON)
```
What do you think about this? That way _EPSILON only has an impact on the results if it needs to. Might not matter at all though.